### DISCUSSION TITLE
EV Demand in ReEDS

### DISCUSSION THREAD

**Discussion by balasubramaniansa** [NREL/ReEDS-2.0#135] [Q&A: Data]
Hi ReEDS Team,
                     I was checking the EV demand in the ReEDS. There are EV_static_demand and EV_dynamic_demand. Can you please tell what is static and dynamic demand? Thank you.

Bala

**Comment by wesleyjcole** [comment]
The static demand is the portion of electric vehicle load that is inflexible.  The dynamic demand is the portion that is flexible and is therefore available by the model to be shifted among timeslices.  This capability was used in some of our earlier EV work such as https://www.nrel.gov/docs/fy17osti/66980.pdf.
